SingularityService/src/test/java/com/hubspot/singularity/data/SingularityValidatorTest.java

```diff
@@ -0,0 +1,63 @@
+package com.hubspot.singularity.data;
+
+import static org.mockito.Mockito.*;
+
+import javax.ws.rs.WebApplicationException;
+
+import org.jukito.JukitoModule;
+import org.jukito.JukitoRunner;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.junit.runner.RunWith;
+
+import com.google.common.base.Optional;
+import com.hubspot.singularity.SingularityDeploy;
+import com.hubspot.singularity.SingularityRequest;
+import com.hubspot.singularity.config.MesosConfiguration;
+import com.hubspot.singularity.config.SingularityConfiguration;
+
+@RunWith(JukitoRunner.class)
+public class SingularityValidatorTest {
+
+  @Rule
+  public ExpectedException expectedException = ExpectedException.none();
+
+  public static class TestModule extends JukitoModule {
```

Comment:
We actually have a class set up that handles most of this setup already. If you just add these tests in the `ValidatorTest` class (feel free to rename to `SingularityValidatorTest` for consistency), that class already has all of the guice bits taken care of and you can use the already injected `SingularityValidator`. Saves us from adding the mockito dep for just this class as well ;)